### PR TITLE
Show and allow editing of min/max values for continuous coloring

### PR DESCRIPTION
**Pull request by juliexxia**
Along the way, fix a bug where if you deleted a layer while editing its coloring, the coloring box stays up with the deleted layers information.

#276

### CODE REVIEW

File: docs/import/color_function_util.js [hunk 1]
```diff
@@ -90,6 +106,58 @@ function setProperty(picker) {
...
+  $('#continuous-max').val(stats['max']);
+}
+
+/**
+ * Validates the given min or max value is valid and writes it.
+ * @param {boolean} min
+ * @param {JQuery<HTMLElement>} continuousPropertyPicker
+ * @return {?Promise<void>} Returns when finished writing or null if it just
+ * queued a write and doesn't know when that will finish. Also returns null
+ * if there the new max or min value was bad.
```
Comment (janakdr) [inline docs/import/color_function_util.js]: nit: "if there the"

File: docs/layer_util.js [hunk 2]
```diff
@@ -382,6 +382,8 @@ function addLayerFromFeatures(layerDisplayData, index) {
         createStyleFunction(deckParams.colorFunctionProperties);
   }
   deckGlArray[index] = new deck.GeoJsonLayer({
+    // TODO: also add layer index to this id so we don't get duplicate ids from
```
Comment (janakdr) [inline docs/layer_util.js]: nit: unrelated? fine with leaving in, though

File: cypress/integration/unit_tests/manage_layers_test.js [hunk 3]
```diff
@@ -198,9 +198,11 @@ describe('Unit tests for manage_layers page', () => {
     const tbody = createAndAppend('tbody', 'tbody');
     const rows = createTrs(2);
     tbody.append(rows);
+    const colorEditor = createAndAppend('div', 'color-fxn-editor').show();
 
     cy.stub(window, 'confirm').returns(true);
     cy.wrap(onDelete(rows[0])).then(() => {
+      expect(colorEditor.is(':visible')).to.be.false;
```
Comment (juliexxia) [inline cypress/integration/unit_tests/manage_layers_test.js]: Added this line here to test the color editor being hidden - lmk if you'd like more

File: docs/layer_util.js (hunk 2 above)
Comment (juliexxia) [reply docs/layer_util.js]: Agreed, just thought of it during this PR.

File: cypress/integration/unit_tests/manage_layers_test.js (hunk 3 above)
Comment (janakdr) [reply cypress/integration/unit_tests/manage_layers_test.js]: Oops, thanks!

**Pull request merged**